app/src/main/java/com/example/robin/roomwordsample/Utils/utils.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.robin.roomwordsample.Utils
+
+import android.content.Context
+import android.view.inputmethod.InputMethodManager
+
+class utils {
```

Comment:
Why didn't you directly declared utils as object instead of making it a class and then making its companion object?

Reply:
Done.